src/components/look-controls.js

```diff
@@ -227,34 +227,27 @@ module.exports.Component = registerComponent('look-controls', {
   /**
    * Handle positional tracking.
    */
-  updatePosition: (function () {
-    var deltaHMDPosition = new THREE.Vector3();
-
-    return function () {
-      var el = this.el;
-      var currentHMDPosition;
-      var currentPosition;
-      var position = this.position;
-      var previousHMDPosition = this.previousHMDPosition;
-      var sceneEl = this.el.sceneEl;
-
-      if (!sceneEl.is('vr-mode')) { return; }
+  updatePosition: function () {
+    var el = this.el;
+    var currentHMDPosition;
+    var currentPosition;
+    var position = this.position;
+    var previousHMDPosition = this.previousHMDPosition;
+    var sceneEl = this.el.sceneEl;
 
-      // Calculate change in position.
-      currentHMDPosition = this.calculateHMDPosition();
-      deltaHMDPosition.copy(currentHMDPosition).sub(previousHMDPosition);
+    if (!sceneEl.is('vr-mode')) { return; }
 
-      if (isNullVector(deltaHMDPosition)) { return; }
+    // Calculate change in position.
+    currentHMDPosition = this.calculateHMDPosition();
 
-      previousHMDPosition.copy(currentHMDPosition);
+    currentPosition = el.getAttribute('position');
 
-      currentPosition = el.getAttribute('position');
-      position.x = currentPosition.x + deltaHMDPosition.x;
-      position.y = currentPosition.y + deltaHMDPosition.y;
-      position.z = currentPosition.z + deltaHMDPosition.z;
-      el.setAttribute('position', position);
-    };
-  })(),
+    position.x = currentPosition.x - previousHMDPosition.x + currentHMDPosition.x;
```

Comment:
Wonder if we can make `this.position` a `Vector3` and use the `sub`/`add` methods instead of doing individual x/y/z operations. Goes for the tracked-controls as well.